docs/api/capp.md

```diff
@@ -38,12 +38,43 @@
     }
     ```
   - **Response**: Capp info or an error message.
-    ```json
-    {
-       "capps": []str,
-       "count": int
+
+```json
+{
+  "capp": {
+    "metadata": {
+      "name": "string",  // max length 53 char
+    },
+    "annotations": [
+      {
+        "key": "string",
+        // Key
+        "value": "string"
+        // Value
+      },
+      {
+        "key": "string",
+        "value": "string"
+      }
+    ],
+    "labels": [
+      {
+        "key": "string",
+        // Key
+        "value": "string"
+        // Value
+      },
+      {
+        "key": "string",
+        "value": "string"
+      }
+    ],
+    "spec": {
+      // [CappSpec] (https://github.com/dana-team/container-app-operator/blob/v0.2.0/api/v1alpha1/capp_types.go#L29
```

Comment:
Link to `main` instead of `v0.2.0` - https://github.com/dana-team/container-app-operator/blob/main/api/v1alpha1/capp_types.go#L30-L62